compiler/fir/checkers/checkers-component-generator/src/org/jetbrains/kotlin/fir/checkers/generator/diagnostics/FirDiagnosticsList.kt

```diff
@@ -168,6 +172,14 @@ object DIAGNOSTICS_LIST : DiagnosticList() {
         }
 
         val VARARG_OUTSIDE_PARENTHESES by error<FirSourceElement, KtExpression>()
+        val NON_VARARG_SPREAD by error<FirSourceElement, LeafPsiElement>()
+        val ARGUMENT_PASSED_TWICE by error<FirSourceElement, KtValueArgument>(PositioningStrategy.NAME_OF_NAMED_ARGUMENT)
```

Comment:
Not `KtReferenceExpression`?

Reply:
So we talked about whether to report the entire argument and use a position strategy to pick up the name or report the name directly. Since we agreed to do the former I am reporting the entire argument. If we want to report the name, I will need to add a util function that locates the name part from an argument.

Reply:
I didn't ask about positioning; it was simply `s/KtValueArgument/KtReferenceExpression/g` (to match with old diagnostic). Does it affect positioning?

Reply:
Duh, I forgot you mentioned this internally. Sorry!

Reply:
Np! Sounds like we are on the same page now ;)